vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -111,68 +101,74 @@ private void openProjectInformationDialog() {
-        personSearchService,
-        project.getProjectIntent().projectTitle(), project.getProjectIntent().objective(),
-        project.getProjectIntent().experimentalDesign(), project.getPrincipalInvestigator(),
-        responsiblePerson, project.getProjectManager());
-    projectInformationDialog.setConfirmButtonLabel("Save");
-    return projectInformationDialog;
-  }
-
-  private void addProjectInformationDialogListeners(ProjectId projectId,
-      ProjectInformationDialog projectInformationDialog) {
-    projectInformationDialog.addCancelEventListener(
-        experimentInformationDialogCancelEvent -> projectInformationDialog.close());
-    projectInformationDialog.addConfirmEventListener(projectInformationDialogConfirmEvent -> {
-      if (projectInformationDialog.isInputValid()) {
-        ProjectInformationContent projectInformationContent = projectInformationDialogConfirmEvent.getSource()
-            .content();
-        updateProjectInformation(projectId, projectInformationContent);
-        projectInformationDialog.close();
-        fireEditEvent();
-      }
-    });
+    projectUpdateEvent.getSource().close();
   }
 
-  private void updateProjectInformation(ProjectId projectId,
-      ProjectInformationContent projectInformationContent) {
-    projectInformationService.updateTitle(projectId, projectInformationContent.projectTitle());
+  private void updateProjectInformation(ProjectInformation projectInformationContent) {
+    ProjectId projectId = this.context.projectId().orElseThrow();
+    projectInformationService.updateTitle(projectId, projectInformationContent.getProjectTitle());
     projectInformationService.stateObjective(projectId,
-        projectInformationContent.projectObjective());
-    projectInformationService.describeExperimentalDesign(projectId,
-        projectInformationContent.experimentalDesignDescription());
+        projectInformationContent.getProjectObjective());
     projectInformationService.investigateProject(projectId,
-        projectInformationContent.principalInvestigator());
+        fromContact(projectInformationContent.getPrincipalInvestigator()));
     projectInformationService.manageProject(projectId,
-        projectInformationContent.projectManager());
-    projectInformationService.setResponsibility(projectId,
-        projectInformationContent.responsiblePerson());
+        fromContact(projectInformationContent.getProjectManager()));
+    projectInformationContent.getResponsiblePerson()
+        .map(ProjectDetailsComponent::fromContact)
+        .ifPresent(person -> projectInformationService.setResponsibility(projectId, person));
   }
 
-  private void addListenerForNewEditEvent() {
-    this.editListeners.add(event -> loadProjectData(event.projectId()));
+  private static Contact fromContact(life.qbic.datamanager.views.general.contact.Contact contact) {
+    return new Contact(contact.getFullName(), contact.getEmail());
```

Comment:
NIT: Maybe the classes could be name distinctly dependent on their context?(Personreference in the domain, contact in the application)